### PR TITLE
Issue #SC-963 chore: Update samza jobs for ingesting LOG events to druid

### PR DESCRIPTION
**Pull request by Sanjay-Krishnan93**
@anandp504 @sowmya-dixit @sunilpes Please review.

### CODE REVIEW

File: data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void setUp() {
         collectorMock = mock(MessageCollector.class);
         contextMock = mock(TaskContext.class);
         metricsRegistry = Mockito.mock(MetricsRegistry.class);
-        counter = Mockito.mock(Counter.class);
+        counter = new Counter("Counter");
```
Comment (anandp504) [inline data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java]: The counter need not be a concrete object.

File: data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java [hunk 2]
```diff
@@ -586,4 +587,40 @@ public boolean matches(Object o) {
         };
     }
 
+    @Test
+    public void shouldSendEventsToSuccessTopicForLogEvents() throws Exception {
+        stub(envelopeMock.getMessage()).toReturn(EventFixture.LOG_EVENT);
```
Comment (anandp504) [inline data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java]: What are we testing here?

File: data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java [hunk 3]
```diff
@@ -586,4 +587,40 @@ public boolean matches(Object o) {
...
+                return true;
+            }
+        }));
+    }
+
+    @Test
+    public void shouldSkipProcessingForErrorEvents() throws Exception {
+        stub(envelopeMock.getMessage()).toReturn(EventFixture.ERROR_EVENT);
+        deNormalizationTask.process(envelopeMock, collectorMock, coordinatorMock);
+        assertEquals(1, counter.getCount());
```
Comment (anandp504) [inline data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java]: Is this a valid assertion here? If it were a LOG event, even then the counter will be incremented by 1.

File: data-pipeline/events-router/src/main/config/events-router.properties [hunk 4]
```diff
@@ -63,6 +63,7 @@ job.coordinator.replication.factor=1
 
 router.events.telemetry.route.topic=__env__.events.telemetry
 router.events.summary.route.topic=__env__.events.summary
+router.events.log.route.topic=__env__.events.log
```
Comment (anandp504) [inline data-pipeline/events-router/src/main/config/events-router.properties]: Have changes been made to create the Kafka topic using ansible?

**Comment by Sanjay-Krishnan93** [comment]
> Please address the PR comments.

Changes has been made as specified.

File: data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java (hunk 1 above)
Comment (Sanjay-Krishnan93) [reply data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java]: Made changes for counter

File: data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java (hunk 2 above)
Comment (Sanjay-Krishnan93) [reply data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java]: Pushing sample LOG event and checking its output values after denormalization.

File: data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java (hunk 3 above)
Comment (Sanjay-Krishnan93) [reply data-pipeline/de-normalization/src/test/java/org/ekstep/ep/samza/task/DeNormalizationTaskTest.java]: Modified the test case

File: data-pipeline/events-router/src/main/config/events-router.properties (hunk 4 above)
Comment (Sanjay-Krishnan93) [reply data-pipeline/events-router/src/main/config/events-router.properties]: Updated ansible scripts

**Pull request merged**